syncweek-react-aad/react-aad
# Add forceLogin parameter to AzureAD

## Purpose
* Resolves issue #83 by providing a forceLogin parameter to the AzureAD component to initiate login for unauthenticated users as soon as the component loads.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Add a `forceLogin={true}` parameter to the redirect sample and verify that (a) the page redirects when unauthenticated, (b) when returning and authenticated, the user is not redirected to login again, and (c) after logging out and returning to the page, the user is forcibly redirected to login.
